packages/bson-transpilers/test/yaml/bson-methods.yaml

```diff
@@ -35,6 +35,10 @@ tests:
                 BSON::CodeWithScope.new('test code', {
                   'x' => 1
                 }).scope
+            rust: |-
+                JavaScriptCodeWithScope { code: "test code", scope: doc! {
```

Comment:
I think we need a `.to_string()` here after `"test code"`

Reply:
Done, and similarly for uses of `Bson::JavaScriptCode()`.